README.md

```diff
@@ -10,6 +10,7 @@ We use Docker to containerize all micro-services and you `just` need to run a fe
 
 - [Rust ^1.68](https://rustup.rs/)
 - [nodejs ^18.15](https://nodejs.org/en)
+- [Typescript](https://www.typescriptlang.org/)
```

Comment:
To avoid mismatch of users global installed TypeScript version I would recommend to put it into `devDependencies` of `package.json`.